1417/en.md

```diff
@@ -0,0 +1,87 @@
+# LOJ 1133 - Array Simulation
+
+First of all, there are some people and you have to send them some mail.
+Now suppose you know that, if you send a mail to the 1st person, he will be able to forward the mail to the 2nd person.
+Here, one can send mail to another only when there is a connection between them.
+Suppose, there is connection between 1st person and 2nd person and there is connection between 2nd and 3rd person but there is no connection between 1st and 3rd person.
+If you want to send mail to 1st person to 3rd person, you have to send it through 2nd person.
+Now in this problem you have to find the first person so that, you may send mail to maximum number of people.
+We may try to understand the problem through an example.
+
+Example:
+1 2 - 1st person can send mail to 2nd person
+2 1 - 2nd person can send mail to 1st person
+5 3 - 5th person can send mail to 3rd person
+3 4 - 3rd person can send mail to 4th person
+4 5 - 4th person can send mail to 5th person
```

Comment:
* Try including a drawing of the graph. https://csacademy.com/app/graph_editor/
* Markdown makes these 5 lines into 1, resulting in poor readability. Use `<br/>` I guess.